ts/metrics/metric.py

```diff
@@ -19,8 +16,16 @@ class Metric(object):
     Class for generating metrics and printing it to stdout of the worker
     """
 
-    def __init__(self, name, value,
-                 unit, dimensions, request_id=None, metric_method=None):
+    def __init__(
+        self,
+        name,
+        value,
+        unit,
+        dimensions,
+        request_id=None,
+        metric_method=None,
+        metric_type=None,
+    ):
```

Comment:
the input only includes:
- metric_name
- metric_type
- unit
- map dimensions to (id, value) // id can be none.

Note: dimension.py also needs to be updated.
- __init__: parameter value default is None.